packages/design-system-react/src/components/icon/.svgrrc.js

```diff
@@ -3,7 +3,6 @@ module.exports = {
   dimensions: false,
   ref: true,
   svgProps: {
-    role: 'img',
```

Comment:
Also did a search for role to confirm all had been removed

![Screenshot 2025-01-08 at 1 48 55 PM](https://github.com/user-attachments/assets/73050a32-4e38-4269-bdec-e47a1cc59866)